framework/topology.cpp

```diff
@@ -796,12 +810,21 @@ void apply_cpuset_param(char *param)
         };
         auto add_to_set = [&](const struct cpu_info &cpu) {
```

Comment:
this is no longer `add_to_set`. maybe `modify_set`?

Reply:
I thought I'd renamed it to `apply_to_set`. Looks like I missed something.